zulipterminal/ui_tools/boxes.py

```diff
@@ -480,6 +497,27 @@ def autocomplete_streams(self, text: str, prefix_string: str
                                     self.view.pinned_streams)
         return matched_data
 
+    def autocomplete_stream_and_topic(self, text: str, prefix_string: str
+                                      ) -> Tuple[List[str], List[str]]:
+        m = re.search(REGEX_AUTOCOMPLETE_STREAM_TOPIC, text)
+
+        stream = m.group(1) if m else ''
+
+        if self.model.is_valid_stream(stream):
+            stream_id = self.model.stream_id_from_name(stream)
+            topic_names = self.model.topics_in_stream(stream_id)
+        else:
+            topic_names = []
+
+        stream_prefix = f"#**{stream}>"
```

Comment:
Is there a reason you aren't using `prefix_string` directly here. On doing a quick check, I don't think we need this new variable.

Reply:
Thanks. I updated the PR to get the required prefix using `prefix_string`